src/frontend/src/features/layouts/components/thread-view/components/assignees-widget/quick-assign-popover/_index.scss

```
@@ -0,0 +1,138 @@
+}
+
+// Visually hidden text used by aria-live regions. Standard recipe — kept
+// out of any utility lib because we use it once.
+.quick-assign-popover__sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.10.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)